app/controllers/idv/base_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Idv
+  class BaseController < ApplicationController
+    def idv_analytics
```

Comment:
This could be a concern or other mixin module rather than a controller. No strong preference.

Reply:
I probably would've gone with a concern first, but I also don't have a strong preference. I think a controller's fine.

Reply:
Yeah, I think I like the idea of a concern as well, since they're more composable and allows us to keep a single base controller (`ApplicationController`). Updated in 02486d291.